### ISSUE TITLE
JSettlers is running very slow on some Android devices

### ISSUE THREAD

**Issue by paulwedeck** [jsettlers/settlers-remake#763]
More information in #747

**Comment by PaweX** [comment]
On PC as well comparing to The Settlers III - (I have Intel i5).
About Android: in my case the game becomes unplayable after 20-30 min with 2-3 players on my LG F60. Also the phone overheats.

I think it also strongly affected by movement of units e.g. soldiers when AI sends them somewhere.

**Comment by paulwedeck** [comment]
Please open a new issue for your Android problem because this issue is about 0.8 fps right from the beginning and therefore a completely different case.
JSettlers will never reach the performance of The Settlers 3 because Java Bytecode is too slow.

> in my case the game becomes unplayable after 20-30 min with 2-3 players on my LG F60.
Do you mean a low framerate or laggy soldier movement etc. ?

**Comment by PaweX** [comment]
> JSettlers will never reach the performance of The Settlers 3 because Java Bytecode is too slow.
You're right, that's why I wasn't fully happy about making this remake on Java

> Do you mean a low framerate or laggy soldier movement etc. 

The whole game freezes. I don't use this phone anymore, but according to this what I remember, you could move the camera. But not fully sure. Anyway the game is running very slow also on my PC after a while of playing. But here is the overall optimization.

> 0.8 fps right from the beginning and therefore a completely different case.

Yes I think that's different case.
